src/devices/hilux.ts

```diff
@@ -9,6 +9,13 @@ const definitions: Definition[] = [
         description: 'Spot 7W',
         extend: [light({colorTemp: {range: [153, 370]}, powerOnBehavior: false})],
     },
+    {
+        zigbeeModel: ['Dimmer-Switch-ZB3.0'],
+        model: 'Dimmer-Switch-ZB3.0',
```

Comment:
Could you update this with the actual model number written on the device?

Reply:
Unfortunately it is installed by a certified electrician a while back and just didn't get around to fix in z2m before now.

I can't see any model number 😭
![20240713_104708](https://github.com/user-attachments/assets/0e9daa46-14d5-4c4a-b86e-36538510239b)

Reply:
Is there anything written on the site of this product?

Reply:
See the pictures here: https://ledpaerer.dk/zigbee-din-rail-dimmer.html?gad_source=1&gclid=CjwKCAjwy8i0BhAkEiwAdFaeGDe8224f_XdUwuEM_axNunYCiFmEvnXcBzuXeaY28oevcG6o5JkiVRoCxGUQAvD_BwE

I can't see check now that it is installed. I remember the diagram but don't remember if anything was written on the opposite side or the back.

Reply:
On the site I see `D160-ZG`